src/luarocks/build.lua

```diff
@@ -385,16 +386,20 @@ function build.build_rockspec(rockspec, opts)
       return name, version
    end   
 
-   if repos.is_installed(name, version) then
-      repos.delete_version(name, version, opts.deps_mode)
+   local dirs, err
+   if not opts.no_install then
+      if repos.is_installed(name, version) then
+         repos.delete_version(name, version, opts.deps_mode)
```

Comment:
Does this mean that `--no-install` actually uninstalls a previously installed copy?

I think I understand why that is (so that the installed one doesn't override the one built in the local folder). Is that the idea?

Reply:
It's `not opts.no_install`. the old rocks is not uninstalled.

I added this option to help my testing workflow. 

Lua-only rocks can be tested w/o `luarocks make`; mixed rocks (C and lua) cannot and need to be "made" first. `--no-install` stops everything after compiling and linking the rock.

Reply:
@dwenegar Ah, got it! Thanks for the clarification!